lib/pka_dev.h

```diff
@@ -167,42 +167,54 @@ typedef struct
 /// PKA Shim structure
 struct pka_dev_shim_s
 {
-    uint64_t            base;             ///< shim base address
-    uint64_t            size;             ///< shim io memory size
+    uint64_t            base;              ///< shim base address
+    uint64_t            size;              ///< shim io memory size
 
-    uint64_t            trng_err_cycle;   ///< TRNG error cycle
+    uint64_t            eip154_mmio_base;  ///< base address for eip154 mmio registers

```

Comment:
Please see comment above regarding a separate structure.

Also I suggest you use eip15_base instead of eip154_mmio_base.

Reply:
Ok, I'll change it.